### PR TITLE
release fixes

### PR DESCRIPTION
**Pull request by FBartos**
fixes: https://github.com/jasp-stats/jasp-test-release/issues/2789
fixes: https://github.com/jasp-stats/jasp-test-release/issues/2782
fixes: https://github.com/jasp-stats/jasp-test-release/issues/2788

### CODE REVIEW

File: inst/qml/qml_components/BayesianMetaAnalysisInference.qml [hunk 1]
```diff
@@ -43,25 +43,49 @@ Section
 			{
 				value: 				"fixed"
 				label: 				qsTr("Fixed effects")
+				onCheckedChanged:	{
+					bayesianMetaAnalysisAdvanced.priorH0FEValue = 0.5
+					bayesianMetaAnalysisAdvanced.priorH1FEValue = 0.5
+					bayesianMetaAnalysisAdvanced.priorH0REValue = 0
+					bayesianMetaAnalysisAdvanced.priorH1REValue = 0
+				}
```
[the remark this answers is not in the document]
Comment (boutinb) [reply inst/qml/qml_components/BayesianMetaAnalysisInference.qml:51]: Oh I see the logic is already implemented in BayesianMetaAnalysisAdvanced.qml, so all this onCheckedChanged can be removed.
I see one difference in the logic: if the value is `constrainedRandom`, it has here the same value as `averaging` (0.25 for every properties), but in the BayesianMetaAnalysisAdvanced, as it will take the default value (0).

File: inst/Upgrades.qml [hunk 2]
```diff
@@ -651,9 +651,9 @@ Upgrades
 
 		ChangeJS
 		{
-			name:		"inferenceOutputScale"
```
Comment (boutinb) [inline inst/Upgrades.qml:654]: Why did you move the name property? I think changeJS should have the name of the option you want to change.

**Review by boutinb — CHANGES_REQUESTED** [review]

File: inst/qml/qml_components/BayesianMetaAnalysisInference.qml (hunk 1 above)
[the remark this answers is not in the document]
Comment (FBartos) [reply inst/qml/qml_components/BayesianMetaAnalysisInference.qml:51]: I did not change this however, it was added in the rebase

File: inst/Upgrades.qml (hunk 2 above)
Comment (FBartos) [reply inst/Upgrades.qml:654]: I also did not change this however, it was added in the rebase. Not sure how that could have happened? I will ask Don tomorrow, maybe he understands.
